proto/scene.proto

```diff
@@ -81,6 +89,14 @@ message MeshedObject {
 }
 
 // Representation of object to render
+// A sceneObject is one of the main things of this format. It describes all the required fields to use if you want to render soemthing.
+// SceneObject contains:
+// - id: defines the id for whole scene.
```

Comment:
`for whole scene` - do you mean something like `which is unique across the scene`?